### ISSUE TITLE
scbalance issue

### ISSUE THREAD

**Issue by wangshuangyi** [yuqcheng/scBalance#1]
Hi @yuqcheng , many thanks for the great tool. However, I got a bit lost and hope you can help me out.  If the training set is too large, which is very challenging for memory, what should I do if I want to save the training set for testing on a different test set?

**Comment by yuqcheng** [comment]
Hi there, thanks so much for your comment. Your point is very helpful. We did not consider re-using the trained network before. We will soon update the software according to your demand and release a new version.

**Comment by wangshuangyi** [comment]
Thanks very much for your help!

**Comment by yuqcheng** [comment]
Hi @wangshuangyi , sorry for the late reply. We have updated our model for the model-saving function. Please see the new tutorial ([Modeling saving and sampling method choosing](https://github.com/yuqcheng/scBalance/blob/main/Tutorial/Model%20saving%20and%20sampling%20method%20selection.ipynb)) for more details. If you have more questions, please feel free to contact me.
